src/lazyload-image.ts

```diff
@@ -62,7 +62,8 @@ function setLoadedStyle(element: HTMLElement) {
     return element;
 }
 
-export function lazyLoadImage(image: HTMLElement, imagePath: string, errorImgPath: string, offset: number) {
+export function lazyLoadImage(image: HTMLElement, imagePath: string, defaultImagePath: string, errorImgPath: string, offset: number) {
+    setImage(image, defaultImagePath);
```

Comment:
What happens if the user has defined `src` (for an image) but not `defaultImage`? eg.

```html
<img src="some-image.jpg" [lazyLoad]="image">
```

That will remove the the `src`-image, right? I think we should wrap the function call inside a if-statement:

```js
if (defaultImagePath) {
  setImage(image, defaultImagePath);
}
```